### ISSUE TITLE
Fail to run spyder

### ISSUE THREAD

**Issue by woony90** [ContinuumIO/anaconda-issues#1340]
Hi. I am new to Python and I need to install Anaconda 4.2.0 (Python 3.5 version) for academic purpose.
However, spyder won't open even being reset. Just for your information, I am using windows10.

I got the following message after typing "spyder --show-console" on anaconda prompt

(C:\Users\woony90\Anaconda3) C:\Users\woony90>spyder --show-console
Traceback (most recent call last):
File "C:\Users\woony90\Anaconda3\lib\site-packages\qtpy_init_.py", line 119, in 
from PySide import version as PYSIDE_VERSION # analysis:ignore
ImportError: No module named 'PySide'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "C:\Users\woony90\Anaconda3\Scripts\spyder-script.py", line 5, in 
sys.exit(spyder.app.start.main())
File "C:\Users\woony90\Anaconda3\lib\site-packages\spyder\app\start.py", line 103, in main
from spyder.app import mainwindow
File "C:\Users\woony90\Anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 49, in 
requirements.check_qt()
File "C:\Users\woony90\Anaconda3\lib\site-packages\spyder\requirements.py", line 39, in check_qt
import qtpy
File "C:\Users\woony90\Anaconda3\lib\site-packages\qtpy_init_.py", line 125, in 
raise PythonQtError('No Qt bindings could be found')
qtpy.PythonQtError: No Qt bindings could be found

Can anybody help me with this issue?

Thanks!

**Comment by ccordoba12** [comment]
Please post the output of

    conda info

and

    conda list

**Comment by woony90** [comment]
(C:\Users\woony90\Anaconda3) C:\Users\woony90>conda info
Current conda install:

               platform : win-64
          conda version : 4.2.13
       conda is private : False
      conda-env version : 4.2.13
    conda-build version : 2.0.2
         python version : 3.5.2.final.0
       requests version : 2.11.1
       root environment : C:\Users\woony90\Anaconda3  (writable)
    default environment : C:\Users\woony90\Anaconda3
       envs directories : C:\Users\woony90\Anaconda3\envs
          package cache : C:\Users\woony90\Anaconda3\pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
            config file : None
           offline mode : False

**Comment by woony90** [comment]
(C:\Users\woony90\Anaconda3) C:\Users\woony90>conda list
# packages in environment at C:\Users\woony90\Anaconda3:
#
_license                  1.1                      py35_1
_nb_ext_conf              0.3.0                    py35_0
alabaster                 0.7.9                    py35_0
anaconda                  custom                   py35_0
anaconda-clean            1.0.0                    py35_0
anaconda-client           1.5.1                    py35_0
argcomplete               1.0.0                    py35_1
astroid                   1.4.7                    py35_0
astropy                   1.2.1               np111py35_0
babel                     2.3.4                    py35_0
backports                 1.0                      py35_0
beautifulsoup4            4.5.1                    py35_0
bitarray                  0.8.1                    py35_1
blaze                     0.10.1                   py35_0
bokeh                     0.12.2                   py35_0
boto                      2.42.0                   py35_0
bottleneck                1.1.0               np111py35_0
bzip2                     1.0.6                    vc14_3  [vc14]
cffi                      1.7.0                    py35_0
chest                     0.2.3                    py35_0
click                     6.6                      py35_0
cloudpickle               0.2.1                    py35_0
clyent                    1.2.2                    py35_0
colorama                  0.3.7                    py35_0
comtypes                  1.1.2                    py35_0
conda                     4.2.13                   py35_0
conda-build               2.0.2                    py35_0
conda-env                 2.6.0                         0
configobj                 5.0.6                    py35_0
console_shortcut          0.1.1                    py35_1
contextlib2               0.5.3                    py35_0
cryptography              1.5                      py35_0
curl                      7.49.0                   vc14_0  [vc14]
cycler                    0.10.0                   py35_0
cython                    0.24.1                   py35_0
cytoolz                   0.8.0                    py35_0
dask                      0.11.0                   py35_0
datashape                 0.5.2                    py35_0
decorator                 4.0.10                   py35_0
dill                      0.2.5                    py35_0
docutils                  0.12                     py35_2
dynd-python               0.7.2                    py35_0
entrypoints               0.2.2                    py35_0
et_xmlfile                1.0.1                    py35_0
fastcache                 1.0.2                    py35_1
filelock                  2.0.6                    py35_0
flask                     0.11.1                   py35_0
flask-cors                2.1.2                    py35_0
freetype                  2.5.5                    vc14_1  [vc14]
get_terminal_size         1.0.0                    py35_0
gevent                    1.1.2                    py35_0
greenlet                  0.4.10                   py35_0
h5py                      2.6.0               np111py35_2
hdf5                      1.8.15.1                 vc14_4  [vc14]
heapdict                  1.0.0                    py35_1
icu                       57.1                     vc14_0  [vc14]
idna                      2.1                      py35_0
imagesize                 0.7.1                    py35_0
ipykernel                 4.5.0                    py35_0
ipython                   5.1.0                    py35_0
ipython_genutils          0.1.0                    py35_0
ipywidgets                5.2.2                    py35_0
itsdangerous              0.24                     py35_0
jdcal                     1.2                      py35_1
jedi                      0.9.0                    py35_1
jinja2                    2.8                      py35_1
jpeg                      8d                       vc14_2  [vc14]
jsonschema                2.5.1                    py35_0
jupyter_client            4.4.0                    py35_0
jupyter_console           5.0.0                    py35_0
jupyter_core              4.2.0                    py35_0
lazy-object-proxy         1.2.1                    py35_0
libdynd                   0.7.2                         0
libpng                    1.6.22                   vc14_0  [vc14]
libtiff                   4.0.6                    vc14_2  [vc14]
llvmlite                  0.13.0                   py35_0
locket                    0.2.0                    py35_1
lxml                      3.6.4                    py35_0
markupsafe                0.23                     py35_2
menuinst                  1.4.1                    py35_0
mistune                   0.7.3                    py35_0
mkl                       11.3.3                        1
mkl-service               1.1.2                    py35_2
mpmath                    0.19                     py35_1
multipledispatch          0.4.8                    py35_0
nb_anacondacloud          1.2.0                    py35_0
nb_conda                  2.0.0                    py35_0
nb_conda_kernels          2.0.0                    py35_0
nbconvert                 4.2.0                    py35_0
nbformat                  4.1.0                    py35_0
nbpresent                 3.0.2                    py35_0
networkx                  1.11                     py35_0
nltk                      3.2.1                    py35_0
nose                      1.3.7                    py35_1
notebook                  4.2.3                    py35_0
numba                     0.28.1              np111py35_0
numexpr                   2.6.1               np111py35_0
numpy                     1.11.1                   py35_1
odo                       0.5.0                    py35_1
openpyxl                  2.3.2                    py35_0
openssl                   1.0.2j                   vc14_0  [vc14]
pandas                    0.18.1              np111py35_0
partd                     0.3.6                    py35_0
path.py                   8.2.1                    py35_0
pathlib2                  2.1.0                    py35_0
patsy                     0.4.1                    py35_0
pep8                      1.7.0                    py35_0
pickleshare               0.7.4                    py35_0
pillow                    3.3.1                    py35_0
pip                       8.1.2                    py35_0
pkginfo                   1.3.2                    py35_0
ply                       3.9                      py35_0
prompt_toolkit            1.0.3                    py35_0
psutil                    4.3.1                    py35_0
py                        1.4.31                   py35_0
pyasn1                    0.1.9                    py35_0
pycosat                   0.6.1                    py35_1
pycparser                 2.14                     py35_1
pycrypto                  2.6.1                    py35_4
pycurl                    7.43.0                   py35_0
pyflakes                  1.3.0                    py35_0
pygments                  2.1.3                    py35_0
pylint                    1.5.4                    py35_1
pyopenssl                 16.0.0                   py35_0
pyparsing                 2.1.4                    py35_0
pyqt                      5.6.0                    py35_1
pytables                  3.2.2               np111py35_4
pytest                    2.9.2                    py35_0
python                    3.5.2                         0
python-dateutil           2.5.3                    py35_0
pytz                      2016.6.1                 py35_0
pywin32                   220                      py35_1
pyyaml                    3.12                     py35_0
pyzmq                     15.4.0                   py35_0
qt                        5.6.2                    vc14_0  [vc14]
qtawesome                 0.3.3                    py35_0
qtconsole                 4.2.1                    py35_2
qtpy                      1.1.2                    py35_0
requests                  2.11.1                   py35_0
rope                      0.9.4                    py35_1
ruamel_yaml               0.11.14                  py35_0
scikit-learn              0.17.1              np111py35_1
scipy                     0.18.1              np111py35_0
setuptools                27.2.0                   py35_1
simplegeneric             0.8.1                    py35_1
singledispatch            3.4.0.3                  py35_0
sip                       4.18                     py35_0
six                       1.10.0                   py35_0
snowballstemmer           1.2.1                    py35_0
sockjs-tornado            1.0.3                    py35_0
sphinx                    1.4.6                    py35_0
spyder                    3.0.2                    py35_0
sqlalchemy                1.0.13                   py35_0
statsmodels               0.6.1               np111py35_1
sympy                     1.0                      py35_0
tk                        8.5.18                   vc14_0  [vc14]
toolz                     0.8.0                    py35_0
tornado                   4.4.1                    py35_0
traitlets                 4.3.0                    py35_0
unicodecsv                0.14.1                   py35_0
vs2015_runtime            14.0.25123                    0
wcwidth                   0.1.7                    py35_0
werkzeug                  0.11.11                  py35_0
wheel                     0.29.0                   py35_0
widgetsnbextension        1.2.6                    py35_0
win_unicode_console       0.5                      py35_0
wrapt                     1.10.6                   py35_0
xlrd                      1.0.0                    py35_0
xlsxwriter                0.9.3                    py35_0
xlwings                   0.10.0                   py35_0
xlwt                      1.1.2                    py35_0
zlib                      1.2.8                    vc14_3  [vc14]


Here they are. Thank you!

**Comment by woony90** [comment]
(C:\Users\woony90\Anaconda3) C:\Users\woony90>conda list
# packages in environment at C:\Users\woony90\Anaconda3:
#
_license                  1.1                      py35_1
_nb_ext_conf              0.3.0                    py35_0
alabaster                 0.7.9                    py35_0
anaconda                  custom                   py35_0
anaconda-clean            1.0.0                    py35_0
anaconda-client           1.5.1                    py35_0
argcomplete               1.0.0                    py35_1
astroid                   1.4.7                    py35_0
astropy                   1.2.1               np111py35_0
babel                     2.3.4                    py35_0
backports                 1.0                      py35_0
beautifulsoup4            4.5.1                    py35_0
bitarray                  0.8.1                    py35_1
blaze                     0.10.1                   py35_0
bokeh                     0.12.2                   py35_0
boto                      2.42.0                   py35_0
bottleneck                1.1.0               np111py35_0
bzip2                     1.0.6                    vc14_3  [vc14]
cffi                      1.7.0                    py35_0
chest                     0.2.3                    py35_0
click                     6.6                      py35_0
cloudpickle               0.2.1                    py35_0
clyent                    1.2.2                    py35_0
colorama                  0.3.7                    py35_0
comtypes                  1.1.2                    py35_0
conda                     4.2.13                   py35_0
conda-build               2.0.2                    py35_0
conda-env                 2.6.0                         0
configobj                 5.0.6                    py35_0
console_shortcut          0.1.1                    py35_1
contextlib2               0.5.3                    py35_0
cryptography              1.5                      py35_0
curl                      7.49.0                   vc14_0  [vc14]
cycler                    0.10.0                   py35_0
cython                    0.24.1                   py35_0
cytoolz                   0.8.0                    py35_0
dask                      0.11.0                   py35_0
datashape                 0.5.2                    py35_0
decorator                 4.0.10                   py35_0
dill                      0.2.5                    py35_0
docutils                  0.12                     py35_2
dynd-python               0.7.2                    py35_0
entrypoints               0.2.2                    py35_0
et_xmlfile                1.0.1                    py35_0
fastcache                 1.0.2                    py35_1
filelock                  2.0.6                    py35_0
flask                     0.11.1                   py35_0
flask-cors                2.1.2                    py35_0
freetype                  2.5.5                    vc14_1  [vc14]
get_terminal_size         1.0.0                    py35_0
gevent                    1.1.2                    py35_0
greenlet                  0.4.10                   py35_0
h5py                      2.6.0               np111py35_2
hdf5                      1.8.15.1                 vc14_4  [vc14]
heapdict                  1.0.0                    py35_1
icu                       57.1                     vc14_0  [vc14]
idna                      2.1                      py35_0
imagesize                 0.7.1                    py35_0
ipykernel                 4.5.0                    py35_0
ipython                   5.1.0                    py35_0
ipython_genutils          0.1.0                    py35_0
ipywidgets                5.2.2                    py35_0
itsdangerous              0.24                     py35_0
jdcal                     1.2                      py35_1
jedi                      0.9.0                    py35_1
jinja2                    2.8                      py35_1
jpeg                      8d                       vc14_2  [vc14]
jsonschema                2.5.1                    py35_0
jupyter_client            4.4.0                    py35_0
jupyter_console           5.0.0                    py35_0
jupyter_core              4.2.0                    py35_0
lazy-object-proxy         1.2.1                    py35_0
libdynd                   0.7.2                         0
libpng                    1.6.22                   vc14_0  [vc14]
libtiff                   4.0.6                    vc14_2  [vc14]
llvmlite                  0.13.0                   py35_0
locket                    0.2.0                    py35_1
lxml                      3.6.4                    py35_0
markupsafe                0.23                     py35_2
menuinst                  1.4.1                    py35_0
mistune                   0.7.3                    py35_0
mkl                       11.3.3                        1
mkl-service               1.1.2                    py35_2
mpmath                    0.19                     py35_1
multipledispatch          0.4.8                    py35_0
nb_anacondacloud          1.2.0                    py35_0
nb_conda                  2.0.0                    py35_0
nb_conda_kernels          2.0.0                    py35_0
nbconvert                 4.2.0                    py35_0
nbformat                  4.1.0                    py35_0
nbpresent                 3.0.2                    py35_0
networkx                  1.11                     py35_0
nltk                      3.2.1                    py35_0
nose                      1.3.7                    py35_1
notebook                  4.2.3                    py35_0
numba                     0.28.1              np111py35_0
numexpr                   2.6.1               np111py35_0
numpy                     1.11.1                   py35_1
odo                       0.5.0                    py35_1
openpyxl                  2.3.2                    py35_0
openssl                   1.0.2j                   vc14_0  [vc14]
pandas                    0.18.1              np111py35_0
partd                     0.3.6                    py35_0
path.py                   8.2.1                    py35_0
pathlib2                  2.1.0                    py35_0
patsy                     0.4.1                    py35_0
pep8                      1.7.0                    py35_0
pickleshare               0.7.4                    py35_0
pillow                    3.3.1                    py35_0
pip                       8.1.2                    py35_0
pkginfo                   1.3.2                    py35_0
ply                       3.9                      py35_0
prompt_toolkit            1.0.3                    py35_0
psutil                    4.3.1                    py35_0
py                        1.4.31                   py35_0
pyasn1                    0.1.9                    py35_0
pycosat                   0.6.1                    py35_1
pycparser                 2.14                     py35_1
pycrypto                  2.6.1                    py35_4
pycurl                    7.43.0                   py35_0
pyflakes                  1.3.0                    py35_0
pygments                  2.1.3                    py35_0
pylint                    1.5.4                    py35_1
pyopenssl                 16.0.0                   py35_0
pyparsing                 2.1.4                    py35_0
pyqt                      5.6.0                    py35_1
pytables                  3.2.2               np111py35_4
pytest                    2.9.2                    py35_0
python                    3.5.2                         0
python-dateutil           2.5.3                    py35_0
pytz                      2016.6.1                 py35_0
pywin32                   220                      py35_1
pyyaml                    3.12                     py35_0
pyzmq                     15.4.0                   py35_0
qt                        5.6.2                    vc14_0  [vc14]
qtawesome                 0.3.3                    py35_0
qtconsole                 4.2.1                    py35_2
qtpy                      1.1.2                    py35_0
requests                  2.11.1                   py35_0
rope                      0.9.4                    py35_1
ruamel_yaml               0.11.14                  py35_0
scikit-learn              0.17.1              np111py35_1
scipy                     0.18.1              np111py35_0
setuptools                27.2.0                   py35_1
simplegeneric             0.8.1                    py35_1
singledispatch            3.4.0.3                  py35_0
sip                       4.18                     py35_0
six                       1.10.0                   py35_0
snowballstemmer           1.2.1                    py35_0
sockjs-tornado            1.0.3                    py35_0
sphinx                    1.4.6                    py35_0
spyder                    3.0.2                    py35_0
sqlalchemy                1.0.13                   py35_0
statsmodels               0.6.1               np111py35_1
sympy                     1.0                      py35_0
tk                        8.5.18                   vc14_0  [vc14]
toolz                     0.8.0                    py35_0
tornado                   4.4.1                    py35_0
traitlets                 4.3.0                    py35_0
unicodecsv                0.14.1                   py35_0
vs2015_runtime            14.0.25123                    0
wcwidth                   0.1.7                    py35_0
werkzeug                  0.11.11                  py35_0
wheel                     0.29.0                   py35_0
widgetsnbextension        1.2.6                    py35_0
win_unicode_console       0.5                      py35_0
wrapt                     1.10.6                   py35_0
xlrd                      1.0.0                    py35_0
xlsxwriter                0.9.3                    py35_0
xlwings                   0.10.0                   py35_0
xlwt                      1.1.2                    py35_0
zlib                      1.2.8                    vc14_3  [vc14]

**Comment by ccordoba12** [comment]
Everything seems right. Now, what's the output of

    qmake -query

?

**Comment by woony90** [comment]
#(C:\Users\woony90\Anaconda3) C:\Users\woony90>qmake -query
#
#
QT_SYSROOT:
QT_INSTALL_PREFIX:C:/Users/woony90/Anaconda3/Library
QT_INSTALL_ARCHDATA:C:/Users/woony90/Anaconda3/Library
QT_INSTALL_DATA:C:/Users/woony90/Anaconda3/Library
QT_INSTALL_DOCS:C:/Users/woony90/Anaconda3/Library/doc
QT_INSTALL_HEADERS:C:/Users/woony90/Anaconda3/Library/include/qt
QT_INSTALL_LIBS:C:/Users/woony90/Anaconda3/Library/lib
QT_INSTALL_LIBEXECS:C:/Users/woony90/Anaconda3/Library/bin
QT_INSTALL_BINS:C:/Users/woony90/Anaconda3/Library/bin
QT_INSTALL_TESTS:C:/Users/woony90/Anaconda3/Library/tests
QT_INSTALL_PLUGINS:C:/Users/woony90/Anaconda3/Library/plugins
QT_INSTALL_IMPORTS:C:/Users/woony90/Anaconda3/Library/imports
QT_INSTALL_QML:C:/Users/woony90/Anaconda3/Library/qml
QT_INSTALL_TRANSLATIONS:C:/Users/woony90/Anaconda3/Library/translations
QT_INSTALL_CONFIGURATION:
QT_INSTALL_EXAMPLES:C:/Users/woony90/Anaconda3/Library/examples
QT_INSTALL_DEMOS:C:/Users/woony90/Anaconda3/Library/examples
QT_HOST_PREFIX:C:/Users/woony90/Anaconda3/Library
QT_HOST_DATA:C:/Users/woony90/Anaconda3/Library
QT_HOST_BINS:C:/Users/woony90/Anaconda3/Library/bin
QT_HOST_LIBS:C:/Users/woony90/Anaconda3/Library/lib
QMAKE_SPEC:win32-msvc2015
QMAKE_XSPEC:win32-msvc2015
QMAKE_VERSION:3.0
QT_VERSION:5.6.2

**Comment by ccordoba12** [comment]
That looks good too :-/

Now what if you run

    jupyter qtconsole

? That runs well or fails for you?

**Comment by woony90** [comment]
It doesn't work either... :(

(C:\Users\woony90\Anaconda3) C:\Users\woony90>jupyter qtconsole
Traceback (most recent call last):
  File "C:\Users\woony90\Anaconda3\Scripts\jupyter-qtconsole-script.py", line 3, in <module>
    import qtconsole.qtconsoleapp
  File "C:\Users\woony90\Anaconda3\lib\site-packages\qtconsole\qtconsoleapp.py", line 60, in <module>
    from qtconsole.qt import QtCore, QtGui
  File "C:\Users\woony90\Anaconda3\lib\site-packages\qtconsole\qt.py", line 23, in <module>
    QtCore, QtGui, QtSvg, QT_API = load_qt(api_opts)
  File "C:\Users\woony90\Anaconda3\lib\site-packages\qtconsole\qt_loaders.py", line 290, in load_qt
    result = loaders[api]()
  File "C:\Users\woony90\Anaconda3\lib\site-packages\qtconsole\qt_loaders.py", line 216, in import_pyqt5
    from PyQt5 import QtCore, QtSvg, QtWidgets, QtGui
ImportError: DLL load failed: %1 is not a valid Win32 application.



I have environment variables(user variables for woony90) as follows:
C:\Users\woony90\Anaconda3
C:\Users\woony90\Anaconda3\Scripts
C:\Users\woony90\Anaconda3\Library\bin

are they also fine? do I have any missing variables?

**Comment by ccordoba12** [comment]
Perhaps you installed a 64 bits version of Anaconda in a 32 bits Windows version.

Please verify that that's not the case.

**Comment by woony90** [comment]
My window version is a 64bits system. :(

windows edition: Windows10 Home
System processor: i5-3317U
Ram: 4GB (3.71GB usable)
system type: 64-bit Operating system, x64-based processor

**Comment by woony90** [comment]
what should I do now? :(

**Comment by ccordoba12** [comment]
I really don't know. I don't understand what happens in your case, sorry.

**Comment by woony90** [comment]
I typed "conda install pyside" to install pyside because the error message included "No module named 'PySide'"
That produced pyside is in conflict with python3.5 which I already installed with anaconda.
Thus, I downgraded it to python3.4 and finally my spyder works well.
It is little sad that I cannot use python3.5 but at least I can use spyder! Maybe I will try it again after the new versions of anaconda is released.

I appreciate your help and hope you have a great day!
